app/Console/Commands/ImportCSV.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\User;
+use App\Contact;
+use Illuminate\Console\Command;
+
+class ImportCSV extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'import:csv {user} {file}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Imports CSV in Google format to user account';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $file = $this->argument('file');
+        $user = User::find($this->argument('user'));
+        $this->info("Importing CSV file $file to user {$user->id}");
+        $row = 0;
+        $imported = 0;
+        if (($handle = fopen($file, "r")) !== FALSE) {
+            while (($data = fgetcsv($handle)) !== FALSE) {
+                $row++;
+                $has_data = false;
+
+                // don't import the columns
+                if ( $row == 1 ) { continue; }
+
+                $contact = new Contact();
+                $contact->account_id = $user->id;
+
+                if ( ! empty( $data[1] ) ) {
```

Comment:
I think the first check should be to verify that either `$data[0]` or `$data[1]` contains a value (Name or given name) because these are the only data that we need to import. If we don't have one of these two values, we can't import the row, period (a contact without a name won't work in the current UI).
Can you make this check? Thanks a lot!